src/slackbot/checkly.spec.ts

```diff
@@ -0,0 +1,251 @@
+import { expect } from "@jest/globals";
+import { WebClient } from "@slack/web-api";
+import dotenv from "dotenv";
+import {
+  summarizeErrorsPrompt,
+  SummarizeErrorsPromptType,
+} from "../prompts/checkly";
+import { generateObject } from "ai";
+import { ChecklyClient } from "../checkly/checklyclient";
+import {
+  fetchCheckResults,
+  summarizeCheckResult,
+  LAST_30_DAYS,
+  last24h,
+} from "../prompts/checkly-data";
+import { CheckResult } from "../checkly/models";
+import { createHeatmap, generateHeatmapPNG } from "../heatmap/createHeatmap";
+import { analyzeImageBufferForPatterns } from "../heatmap/analyseHeatmap";
+
+dotenv.config();
+
+describe("Checkly Slack Message Tests", () => {
```

Comment:
it's more of a script than a test, right? Do we need this in the test pipeline?

Reply:
will migrate this in my next MR into a jupyter notebook